contributors/devel/sig-instrumentation/migration-to-structured-logging.md

```diff
@@ -7,6 +7,61 @@ should be applicable at later stages or for other projects using `klog` logging
+
+Non-organized migration is done by individual contributors that want particular code section to utilize new features early. Migration of Kubernetes components not yet marked for migration is on best effort bases. This means that we don't guarantee a timely review or that PR will be accepted at all. It's up to component owners to decide if they want to accept those contributions. As this migration is not organized it means that two contributors could start working on migrating same code. Before sending a PR please make sure that there is was already created by someone else.
+
+### Current status
+
+* 1.21 Kubelet was migrated
+* 1.22 We are collecting feedback and making improvements to the migration process.
+
+## Sending a Structured Logging Migration Pull Request
+
+### Before creating the Pull Request
+
+* Read and understood the. In case of any questions, please contact us on [#wg-structured-logging](https://kubernetes.slack.com/messages/wg-structured-logging) Slack channel.
+* Check list of [opened PRs](https://github.com/kubernetes/kubernetes/pulls?q=is%3Aopen+label%3Awg%2Fstructured-logging+is%3Apr) to check if someone is already working on this.
+
+### What to include in the Pull request
+
+To make sure PR is reviewed in timely manner we require authors to provide necessary information for it to be properly triaged and reviewed by knowledgeable parties.
+
+* title: `Migrate <directory/file> to structured logging`
+* WGs: `structured-logging`
+* area: `logging`
+* priority: `important-longterm` for normal PR or `important-soon` for PRs that are part of organized migration of component. 
+* cc: @kubernetes/wg-structured-logging-reviews
+* kind: cleanup
+* release note: `Migrate <directory/file> to structured logging
+`
+To quickly add this information you can comment on PR with below commands:
+
+/retitle Migrate <directory/file> to structured logging
+/wg structured-logging 
+/area logging
+/priority important-longterm
+/king cleanup
+/cc @kubernetes/wg-structured-logging-reviews
+
+And edit the top comment to include release note:
+Migrate <directory/file> to structured logging
+
+### Why my PR was rejected?
+
+Fact that Kubernetes is organizing migration of to Structured Logging doesn't mean that are accepting all the PRs that migrate logs. We reserve the right to reject the Pull Request in situations listed below:
```

Comment:
Reworded:

Even though the Kubernetes project is organizing migration of Structured Logging, this doesn't mean that we are able to accept all the PRs that come our way. We reserve the right to reject the Pull Request in situations listed below: